src/Domain/CoordinateMaps/Wedge3D.cpp

```diff
@@ -141,6 +141,8 @@ std::array<std::decay_t<tt::remove_reference_wrapper_t<T>>, 3> Wedge3D::inverse(
       physical_y = x[2];
       physical_z = -x[0];
     }
+  } else {
+    ERROR("Improper axis passed to function.");
```

Comment:
I don't understand the error message; I don't understand why you added this else-clause; should we use a switch instead since Direction<3>::Axis is an enum?

Reply:
Otherwise clang-tidy will complain about a possible branch where physical_z, used in the denominator for cap_xi, is not initialized
I will try out using a switch